interpreter/cling/test/Pragmas/add_env_path.C

```diff
@@ -0,0 +1,24 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+// RUN: mkdir -p %T/subdir && clang -shared %S/call_lib.c -o %T/subdir/libtest%shlibext
+// RUN: cd %T/subdir && %S/create_header.sh && cd %S
+// RUN: export ENVVAR="%T/subdir"
+// RUN: cat %s | %cling -I %S -Xclang -verify 2>&1 | FileCheck %s
```

Comment:
In principle, the test driver can run each invocation separately, or at least in separate shells. It's better to do
```
export ENVVAR="%T/subdir"; cat %s | %cling -I %S -Xclang -verify 2>&1 | FileCheck %s
```
i.e. all in one line.